src/locales/findLangByUrl.ts

```diff
@@ -0,0 +1,13 @@
+import Lang from './Lang';
+
+const findLangByUrl = (path: string, defaultLang: Lang): Lang.nb | Lang.en => {
```

Comment:
Dette er det vel støtte for i react-router allerede? useParams om jeg husker rett...